daemon/api_debug.go

```diff
@@ -329,6 +343,8 @@ func postDebug(c *Command, r *http.Request, user *auth.UserState) Response {
 		}
 		st.Prune(opTime, 0, 0, 0)
 		return SyncResponse(true, nil)
+	case "create-recovery":
+		return createRecovery(st, a.Params.RecoverySystemLabel)
```

Comment:
Shouldn't this be restricted to root only?

Reply:
This is a handler for POST, for which *OK flags do not apply.

Reply:
this should be fine once  https://github.com/snapcore/snapd/pull/10311 lands